src/ui/server/helpers/sanitise-data/index.ts

```diff
@@ -17,9 +22,9 @@ const {
  * @param {String | Number} Field value
  * @returns {Boolean}
  */
-const shouldChangeToNumber = (value: string | number) => {
-  if (typeof value === 'string' && isNumber(value)) {
-    return true;
+const shouldChangeToNumber = (key: string, value: string | number) => {
+  if (key === COMPANY_NUMBER || key === PHONE_NUMBER || value === '') {
```

Comment:
could use `isEmptyString` here

Reply:
Great shout, created and added new helper function